# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/ia64.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

../sysdeps/ia64/elf/start.S: Assembler messages:
../sysdeps/ia64/elf/start.S:65: Error: missing ')'
../sysdeps/ia64/elf/start.S:65: Error: Illegal operand separator `('

which is due __IA64_UL, ... so let's include the suitable header, too.

  - Rene Rebe <rene@exactcode.de>

--- glibc-2.11.2/sysdeps/ia64/elf/start.S.vanilla	2010-09-23 23:07:30.955431843 +0000
+++ glibc-2.11.2/sysdeps/ia64/elf/start.S	2010-09-23 23:08:09.310962309 +0000
@@ -37,6 +37,7 @@
 #include <sysdep.h>
 
 #include <asm/unistd.h>
+#include <asm/types.h>
 #include <asm/fpu.h>
 
 /*
